### PR TITLE
Add awscli

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
In scope of this pull request we added the following changes:
- added Install-awscli.ps1 and Validate-awscli.ps1 script files to  `images/win/scripts/Installers/`

### CODE REVIEW

File: images/win/scripts/Installers/Validate-Awscli.ps1 [hunk 1]
```diff
@@ -0,0 +1,26 @@
+################################################################################
+##  File:  Validate-awscli.ps1
+##  Desc:  Validate awscli
+################################################################################
+
+$command = Get-Command -Name 'awscli'
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-Awscli.ps1]: There is no awscli.exe for windows. You should use `aws`.

File: images/win/scripts/Installers/Validate-Awscli.ps1 [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+else
+{
+    Write-Host 'awscli is not on path'
+    exit 1
+}
+
+# Adding description of the software to Markdown
+$SoftwareName = "Aws CLI"
+$version = $command.Version.ToString()
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-Awscli.ps1]:
```
PS > (Get-Command -Name 'aws').Version.ToString()
0.0.0.0
```

Parse awscli version from `aws --version` output:
`aws-cli/2.0.5 Python/3.7.5 Windows/10 botocore/2.0.0dev9`

Comment (maxim-lobanov) [reply images/win/scripts/Installers/Validate-Awscli.ps1]: Let's cut this line to take only `aws-cli 2.0.5`

File: images/win/scripts/Installers/Validate-Awscli.ps1 [hunk 3]
```diff
@@ -0,0 +1,27 @@
+################################################################################
+##  File:  Validate-awscli.ps1
+##  Desc:  Validate awscli
+################################################################################
+
+$command = Get-Command -Name 'aws'
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-Awscli.ps1]: Why do you mix case notation?

**Comment by miketimofeev** [comment]
I'd prefer to use 2 separate files actually(Install and Validate), but it's up to you

**Pull request merged**